pkgs/applications/audio/shuriken/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qmake, alsaLib, jack2, rubberband, aubio, libsndfile
+, liblo, libsamplerate }:
+
+stdenv.mkDerivation rec {
+  name = "shuriken-${version}";
```

Comment:
Using `pname = "shrukin";` is now the preferred way to set the application name because `name = "${pname}-${version}";` is implied.